cmd/kubectl-testkube/configure.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+if [ -n "$KUBECONFIG_FILE" ]; then
+	echo "copying KUBECONFIG_FILE to /tmp/kubeconfig/config and exporting KUBECONFIG"
+	$KUBECONFIG_FILE > /tmp/kubeconfig/config
+	export KUBECONFIG=/tmp/kubeconfig/config
+fi
+
+if [ -n "$NAMESPACE" ]; then
+	echo "setting the context for testkube to namespace $NAESPACE"
+	testkube set context --kubeconfig --namespace $NAMESPACE
+elif [ -n "$TESTKUBE_API_KEY" ] && [ -n "$TESTKUBE_ORG_ID" ] && [ -n "TESTKUBE_ENV_ID" ]; then
```

Comment:
Probably worth to also inform User that one/two of variables are missing for fully working Pro